src/Mollie/WC/Payment/Order.php

```diff
@@ -202,7 +202,7 @@ public function getPaymentRequestData( $order, $customer_id ) {
             $paymentRequestData['payment']['cardToken'] = $cardToken;
         }
 
-        if($_POST['token']){
+        if(isset($_POST['token'])){
```

Comment:
I think we don't need this condition at all and it's safe to just remove this line.

 On the next line the `filter_input()` function is called for this $_POST element. If it doesn't exist `null` will be [returned](https://www.php.net/manual/en/function.filter-input.php#refsect1-function.filter-input-returnvalues) without any errors. Even if this filter will fail, this function will return `false`. Both cases are covered by the condition on the line after the next one.